ambry-store/src/main/java/com/github/ambry/store/DiskManager.java

```diff
@@ -186,6 +200,10 @@ void start() throws InterruptedException {
       compactionManager.enable();
       reportUnrecognizedDirs();
       running = true;
+      if(diskHealthCheck.enabled) {
```

Comment:
please expose an isEnabled method from diskHealthCheck object.

Reply:
Created and utilized the isEnabled function